template/typescript/base/package.json

```
@@ -0,0 +1,29 @@
+{
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "serve": "vite preview",
+    "build": "vite build",
+    "lint": "eslint . --fix --ignore-path .gitignore",
+    "dev": "vite"
```

using same commands as in default vite project could be IMHO better
```suggestion
    "dev": "vite",
    "build": "vue-tsc --noEmit && vite build",
    "preview": "vite preview",
    "lint": "eslint . --fix --ignore-path .gitignore"
```